natsumi/ui-tweaks.css

```diff
@@ -199,10 +199,17 @@ menulist[open="true"] menupopup {
           display: none;
         }
       }
-      
+
       .zen-workspace-name {
         color: color-mix(in srgb, var(--toolbarbutton-icon-fill-attention) 70%, light-dark(black, white)) !important;
       }
     }
   }
 }
+
+/* Elongate the pinned tabs section */
+#vertical-pinned-tabs-container {
+  @media (-moz-bool-pref: "natsumi.sidebar.pinned-tabs") {
```

Comment:
Maybe something like `natsumi.sidebar.unlimited-pinned-tabs` would be a better variable name. If you change this, please also change it in the README.

Reply:
Done in the next commit. Will squash as a single commit - so that it's easier for you to merge/rebase into the main branch!

Reply:
(also rebased with your fork's `main` branch with this commit)